### DISCUSSION TITLE
be able to specify what edit windows/websites should use markdown

### DISCUSSION THREAD

**Discussion by danielb2** [glacambre/firenvim#1557] [Ideas]
There's a list here already https://github.com/glacambre/firenvim/blob/138424db463e6c0e862a05166a4ccc781cd7c19d/autoload/firenvimft.vim#L2-L13
of websites that take markdown. Maybe make this accessible via a global var? or is this accessible already?

in addition, it would be cool if I could put 
```
<textarea content-type="text/markdown">
```
in my own textareas to encourage firenvim to use it. but maybe that's asking too much

**Comment by glacambre** [comment]
Hi :)

You can already set arbitrary filetypes using the BufNew/BufEnter command and the name of the buffer, this is described [here](https://github.com/glacambre/firenvim#using-different-settings-depending-on-the-urlpageelement-being-edited).

> it would be cool if I could put <textarea content-type="text/markdown"> in my own textareas to encourage firenvim to use it. but maybe that's asking too much

That's a pretty good idea! I didn't know that a textarea's content type could be set that way, I'll try to implement this :)

**Reply by danielb2** [reply]
You can set any attribute in html but It's not a standard by any means. At least not that I know 😅
